src/app/components/CompletePage/CompletePage.tsx

```diff
@@ -1,12 +1,16 @@
-import React, { useState } from 'react'
+import React from 'react'
 import Expense from '../Expense/Expense'
 import TransactionForm from '../TransactionForm/TransactionForm'
 
 import type { TransactionProps } from '../TransactionForm/TransactionForm'
 import TransactionOverview from '../TransactionOverview/TransactionOverview'
+import useLocalStorage from '../../hooks/useLocalStorage'
 
 export default function CompletePage(): JSX.Element {
-  const [transactions, setTransactions] = useState<TransactionProps[]>([])
+  const [transactions, setTransactions] = useLocalStorage<TransactionProps[]>(
```

Comment:
`TransactionProps` should not be imported from the `TransactionForm`. It looks like they are an essential type for the data you're handling in several components or on your pages. You should declare them in a `types.ts` in your `src` folder

Reply:
I forgot to mention that. thank you